tests/robot-tests/tests/libs/create_emulator_release_files.py

```diff
@@ -162,37 +166,60 @@ def create_public_release_files(self):
             container_client = self.blob_service_client.create_container(
                 "downloads")
         except ResourceExistsError:
+            pass
+
             container_client = self.blob_service_client.get_container_client(
                 "downloads")
 
-        self.upload_files(container_client,
-                          self.all_files_zip_files)
-        self.upload_files(container_client,
-                          self.data_files)
+        try:
+            self.upload_files(container_client,
+                              self.all_files_zip_files)
+
+        except ResourceExistsError:
+            pass
+
+        try:
+            self.upload_files(container_client,
+                              self.data_files)
+
+        except ResourceExistsError:
+            pass
 
     def create_private_release_files(self):
         try:
+
             container_client = self.blob_service_client.create_container(
                 "releases")
+
         except ResourceExistsError:
+            pass
+
             container_client = self.blob_service_client.get_container_client(
                 "releases")
 
-        self.upload_files(container_client,
-                          self.data_files)
-        self.upload_files(container_client,
-                          self.metadata_files)
+        try:
+            self.upload_files(container_client,
+                              self.data_files)
+
+        except ResourceExistsError:
+            pass
+
+        try:
+            self.upload_files(container_client,
+                              self.metadata_files)
 
-    def upload_files(self, container_client, files):
-        data = b'abcd'*128
+        except ResourceExistsError:
+            pass
+
+    @staticmethod
+    def upload_files(container_client, files):
+        data = b'abcd' * 128
         for file in files:
             path = file.path()
-            blob_client = container_client.get_blob_client(path)
-            blob_client.upload_blob(
-                data, blob_type="BlockBlob", metadata=file.metadata())
 
-
-if __name__ == '__main__':
-    generator = ReleaseFilesGenerator()
-    generator.create_public_release_files()
-    generator.create_private_release_files()
+            blob_client = container_client.get_blob_client(path)
+            try:
+                blob_client.upload_blob(
+                    data, blob_type="BlockBlob", metadata=file.metadata())
+            except ResourceExistsError:
+                pass
```

Comment:
I'd print something here - if not the entire error, then just a string to indicate that a file hasn't been uploaded.